python/exercise2/time_window.py

```diff
@@ -20,12 +20,20 @@ def __init__(self, start_time, end_time, delta):
                 
         """
         
+        # Error checking of delta input
```

Comment:
The error messages here don't match up. For the delta >23 and delta<0 cases, the error message accepts 0 hours as a valid delta, but the error message in the delta==0 case says the opposite. Recommend editing the error message to reflect the expected range of values.